iOS_SDK/OneSignalSDK/Source/OSInAppMessagingDefines.h

```diff
@@ -73,6 +73,7 @@ typedef NS_ENUM(NSUInteger, OSTriggerOperatorType) {
 #define OS_IAM_SEEN_SET_KEY @"OS_IAM_SEEN_SET"
 #define OS_IAM_CLICKED_SET_KEY @"OS_IAM_CLICKED_SET"
 #define OS_IAM_IMPRESSIONED_SET_KEY @"OS_IAM_IMPRESSIONED_SET"
+#define OS_IAM_SEEN_WITH_REDISPLAY_DICTIONARY_KEY @"OS_IAM_SEEN_WITH_REDISPLAY_DICTIONARY_KEY"
```

Comment:
Remove `_KEY` from end of `String` `@"OS_IAM_SEEN_WITH_REDISPLAY_DICTIONARY_KEY"`